documentation/src/main/asciidoc/stories/assembly_cache_configuration.adoc

```diff
@@ -10,6 +10,25 @@ For instance, you can configure {brandname} clusters to use replicated caches or
 Your configuration also defines the characteristics of your caches and enables the {brandname} capabilities that you want to use when handling data.
 For instance, you can configure how {brandname} encodes entries in your caches, whether replication requests happen synchronously or asynchronously between nodes, if entries are mortal or immortal, and so on.
 
+[IMPORTANT]
+====
+{brandname} places a restriction of less than or equal to 255 characters on a cache name or a cache template name.
+
+If you exceed this character limit, {brandname} throws an exception.
+
+Try to write succinct cache names and cache template names.
+====
+
+[IMPORTANT]
+====
+A local host's file system might place a character limitation on a file name.
+Check the file system's configuration for any specified character limitations, and ensure that your file name does not exceed this limitation. 
+
+If your file name exceeds a file system's character limitation, the write operation for adding a file name to the file system might fail.
+
+Try to write succinct file names.
```

Comment:
We should rephrase this a little. Something like:

The file system may impose limitations on the length of a file name, so ensure that a cache name does not exceed this limitation. If the cache name exceeds the file system limitations, operations and/or initialization might fail.

Reply:
Thanks, @jabolina .

I updated the content based on your feedback. 

Would it be fair to state that initialization is a type of operation? 

I stated "general operations or initialing operations towards that cache might fail", because numerous entities might perform operations on a cache. Is this accurate?

Reply:
I think it should be OK.